File: Java_ChildrenCompetition/src/main/java/model/Participant.java
package model;

import repository.HasId;

public class Participant implements HasId<Integer> {
    private int participantID;
    private final String name;
    private final int age;

    public Participant(int participantID, String name, int age) {
        this.participantID = participantID;
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    @Override
    public String toString() {
        return "Participant{" +
                "participantID=" + participantID +
                ", name='" + name + '\'' +
                ", age=" + age +
                '}';
    }

    @Override
    public Integer getId() {
        return participantID;
    }

    @Override
    public void setId(Integer id) {
        this.participantID = id;
    }
}

File: CSharp_ChildrenCompetition/CSharp_ChildrenCompetition/CSharp_ChildrenCompetition/model/Participant.cs
// using System;
using CSharp_ChildrenCompetition.repository;

namespace CSharp_ChildrenCompetition.model
{
    public class Participant : HasId<int>
    {
        public int id { get; set; }
        public string name { get; }
        public int age { get; }

        public Participant(int participantId, string name, int age)
        {
            this.id = participantId;
            this.name = name;
            this.age = age;
        }

        public override string ToString()
        {
            return "Name: " + this.name + " age: " + this.age;
        }
    }
}